Assets/Scripts/Singleton.cs

```diff
@@ -0,0 +1,19 @@
+﻿using UnityEngine;
```

Comment:
Singleton class that the GameManager inherits from. Makes sure that we only have one instance of GameManager